string data
cobots_core/Version tag
---
bool status
string message
